src/Jackett.Common/Definitions/mma-torrents.yml

```diff
@@ -0,0 +1,164 @@
+      - {id: 91, cat: TV/Sport, desc: "Packs - Other/HD"}
+      - {id: 64, cat: TV/Sport, desc: "Packs - Other/SD"}
+      - {id: 95, cat: TV/Sport, desc: "Packs - Pride"}
+      - {id: 59, cat: TV/Sport, desc: "Packs - TUF"}
+      - {id: 114, cat: TV/Sport, desc: "Packs - UFC"}
+      - {id: 122, cat: TV/Sport, desc: "Packs - Userpacks"}
+      - {id: 72, cat: TV/Sport, desc: "Retro - K-1"}
+      - {id: 61, cat: TV/Sport, desc: "Retro - Other/All"}
+      - {id: 66, cat: TV/Sport, desc: "Retro - PRIDE"}
+      - {id: 67, cat: TV/Sport, desc: "Retro - UFC"}
+      - {id: 69, cat: TV/Sport, desc: "Retro - WEC"}
+      - {id: 97, cat: TV/Sport, desc: "TUF - Episode/HD"}
+      - {id: 104, cat: TV/Sport, desc: "TUF - Episode/SD"}
+      - {id: 101, cat: TV/Sport, desc: "TV Show - Episode/HD"}
+      - {id: 94, cat: TV/Sport, desc: "TV Show - Episode/SD"}
+      - {id: 127, cat: TV/Sport, desc: "TV Show - WebRip"}
+
+    modes:
+      search: [q]
+      tv-search: [q, season, ep]
+
+  settings:
+    - name: username
+      type: text
+      label: Username
+    - name: password
+      type: password
+      label: Password
+    - name: sort
+      type: select
+      label: Sort requested from site
+      default: "seed"
+      options:
+        "name": "name"
+        "size": "size"
+        "seed": "seeders"
+    - name: type
+      type: select
+      label: Order requested from site
+      default: "desc"
+      options:
+        "desc": "desc"
+        "asc": "asc"
+
+  login:
+    path: account-login.php
+    method: post
+    inputs:
+      username: "{{ .Config.username }}"
+      password: "{{ .Config.password }}"
+      returnto: "/"
+    error:
+      - selector: table.main:contains("Login failed!")
+        message:
+          selector: table tr td.text
+    test:
+      path: torrents.php
+      selector: a[href="account-logout.php"]
+
+  search:
+    paths:
+      - path: torrents.php
+    inputs:
+      $raw: "{{ range .Categories }}c{{.}}=1&{{end}}"
+      incldead: 1
+      search: "{{ .Keywords }}"
+      sort: "{{ .Config.sort }}"
+      type: "{{ .Config.type }}"
+
+    rows:
+      selector: table[class="ttable_headinner"] tr:has(a[href^="download.php?id="])
+
+    fields:
+      category:
+        selector: a[href^="torrents.php?cat="]
+        attribute: href
+        filters:
+          - name: querystring
+            args: cat
+      title:
+        selector: a[href^="torrents-details.php?id="]
+        attribute: title
+      details:
+        selector: a[href^="torrents-details.php?id="]
+        attribute: href
+      comments:
+        selector: a[href^="comments.php?type=torrent&id="]
+        attribute: href
+      download:
+        selector: a[href^="download.php?id="]
+        attribute: href
+      size:
+        selector: td:nth-child(6)
+        filters:
+          - name: trim
+      grabs:
+        optional: true
+        selector: td:nth-child(2)
+        filters:
```

Comment:
very unusual for the grabs to be on the same selector as the date in line153.
if this is really the case then the date will most likely need a regexp to extract the date-data before dateparse is used.
Also rather unusual for the date to be optional... on most sites the date is either not provided or always available?

Reply:
I was able to get the search to render with using dateparse, but different searches would eventually cause a parsing error, but somehow still returned all of the correct sections